.github/workflows/gradle.yml

```diff
@@ -96,7 +96,11 @@ jobs:
         run: |
           mkdir -p ~/.gradle/
           cat > ~/.gradle/gradle.properties <<EOF
-          org.gradle.jvmargs=-Xmx8g -Xss4m
+          org.gradle.jvmargs=-Xmx8g -Xss4m --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
```

Comment:
what is this for?

Reply:
We already have this in the root `gradle.properties` from https://github.com/airbytehq/airbyte/pull/7104 

Spotless doesn't work without these settings unfortunately.

Reply:
The real purpose of this override `gradle.properties` file is to increase RAM usage. from `4g` to `8g`. We could change this to copy the root file and replace `4g` with `8g` instead of redeclaring everything.